samza-core/src/main/java/org/apache/samza/operators/spec/OperatorSpec.java

```diff
@@ -79,10 +97,30 @@ public void registerNextOperatorSpec(OperatorSpec<OM, ?> nextOperatorSpec) {
     nextOperatorSpecs.add(nextOperatorSpec);
   }
 
+  /**
+   * Get the collection of chained {@link OperatorSpec}s that are consuming the output of this node
+   *
+   * @return the collection of chained {@link OperatorSpec}s
+   */
   public Collection<OperatorSpec<OM, ?>> getRegisteredOperatorSpecs() {
     return nextOperatorSpecs;
   }
 
+  public static byte[] toBytes(OperatorSpec opSpec) throws IOException {
```

Comment:
This is the same logic as the `SerializableSerde`. We can use that instead of these methods.